kermit-core/src/darwinMain/kotlin/co/touchlab/kermit/OSLogWriter.kt

```diff
@@ -49,6 +51,7 @@ open class OSLogWriter internal constructor(
         }
     }
 
+    @OptIn(ExperimentalNativeApi::class)
     open fun logThrowable(osLogSeverity: UByte, throwable: Throwable){
         darwinLogger.log(osLogSeverity, throwable.getStackTrace().joinToString("\n"))
     }
```

Comment:
I'm a little paranoid about binary compatibility breakage with some of these opt-ins, but it's probably safe enough since I don't think we expose any experimental types in our public API